corehq/apps/hqcase/tests/test_case_update_api.py

```diff
@@ -0,0 +1,437 @@
+import uuid
+from datetime import datetime
+from unittest import skip
+
+from django.test import TestCase
+from django.urls import reverse
+
+from casexml.apps.case.mock import CaseBlock, IndexAttrs
+from couchforms.models import XFormError
+
+from corehq import privileges
+from corehq.apps.domain.shortcuts import create_domain
+from corehq.apps.users.models import WebUser
+from corehq.form_processor.interfaces.dbaccessors import (
+    CaseAccessors,
+    FormAccessors,
+)
+from corehq.form_processor.tests.utils import FormProcessorTestUtils
+from corehq.util.test_utils import privilege_enabled
+
+from ..api import serialize_case
+from ..utils import submit_case_blocks
+
+
+@privilege_enabled(privileges.API_ACCESS)
+class TestCaseAPI(TestCase):
+    domain = 'test-update-cases'
+    maxDiff = None
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.domain_obj = create_domain(cls.domain)
+        cls.web_user = WebUser.create(cls.domain, 'netflix', 'password', None, None)
+        cls.web_user.is_superuser = True  # in pre-release, this is superuser-only
+        cls.web_user.save()
+        cls.case_accessor = CaseAccessors(cls.domain)
+        cls.form_accessor = FormAccessors(cls.domain)
+
+    def setUp(self):
+        self.client.login(username='netflix', password='password')
+
+    def tearDown(self):
+        FormProcessorTestUtils.delete_all_cases(self.domain)
```

Comment:
Those should get cleaned up with the domain deletion, but I guess since forms are submited in tests, they should be cleaned up here too.  Will add, thanks

Reply:
Makes sense that they will be cleaned up in `tearDownClass`. Probably no need to do it here then.